backend/src/controllers/auth/index.ts

```diff
@@ -1,36 +1,51 @@
 import { Request, Response } from "express";
 import { check, validationResult } from "express-validator";
+import db from "../../config/db-config";
 
-const poolData = {
-    UserPoolId: process.env.COGNITO_USER_POOL_ID || '',
-    ClientId: process.env.COGNITO_APP_CLIENT_ID || ''
-};
+const signUp = async (req: Request, res: Response) => {
```

Comment:
I assume you no longer need these changes since you would be making auth process on the FE

Reply:
I was thinking of linking the DB also and when we have success signup in firebase then send the user to db.
With type user or admin , what do you think with this approach?

Reply:
It makes sense, but for it, I assume, you would need to have another route, like /create-user or something like that, because it has no logic of a sign-in